.github/workflows/community.yml

```
@@ -20,8 +20,9 @@ jobs:
       - uses: EddieJaoudeCommunity/gh-action-community/src/welcome@main
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          issue-message: '<h1>It''s great having you contribute to this project:tada:</h1> Feel free to raise an <strong>Issue</strong>! Welcome to the community :nerd_face:'
-          pr-message: '<h1>It''s great having you contribute to this project:tada:</h1> Feel free to create a <strong>Pull Request</strong>! Welcome to the community :nerd_face:'
+          issue-message: '<h1>It''s great having you contribute to this project</h1> Welcome to the community :nerd_face:'
+          pr-message: '<h1>It''s great having you contribute to this project</h1> Welcome to the community :nerd_face:'
+          footer: 'If you would like to continue contributing to open source and would like to do it with an awesome inclusive community, you should join our <a href="http://discord.eddiehub.org">Discord</a> chat and our <a href="http://github.eddiehub.org">GitHub Organisation</a> - we help and encourage each other to contribute to open source little and often 🤓 . Any questions let us know.'
```

Could we use the verbiage- Discord Server and but the hyperlink on Discord Server for consistency?
```suggestion
          footer: 'If you would like to continue contributing to open source and would like to do it with an awesome inclusive community, you should join our <a href="http://discord.eddiehub.org">Discord Server</a> and our <a href="http://github.eddiehub.org">GitHub Organisation</a> - we help and encourage each other to contribute to open source little and often 🤓 . Any questions let us know.'
```